src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -509,7 +509,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ItemGroup>
       <_BlazorPublishBootResource
         Include="@(ResolvedFileToPublish)"
-        Condition="$([System.String]::Copy('%(RelativePath)').Replace('\','/').StartsWith('wwwroot/_framework')) AND '%(Extension)' != '.a'" />
+        Condition="$([System.String]::Copy('%(RelativePath)').Replace('/','\').StartsWith($(_BlazorFrameworkPublishPath))) AND '%(Extension)' != '.a'" />
```

Comment:
> .Replace('/','\')

What's the reasoning for this bit of change?

Reply:
It's likely because we normalized things earlier within ResolvedFileToPublish and that broke this target.